modules/ROOT/pages/twilio/twilio-connector.adoc

```diff
@@ -172,7 +172,7 @@ output application/java
 {
     Body: "You are now subscribed!",
     From: "${fromNumber}",
-    To: "+" ++ inboundProperties.'http.uri.params'.toNumber
+    To: "+" ++ attributes.uriParams.toNumber default "my default value"
 }
 ----
 +
```

Comment:
HIDDEN TEXT
Edited, please compare

. Add a Twilio Connector after the transform message and apply the following settings:
** Select the Send Message operation.
** Set Account Sid to `${accountSid}`, and Entity Reference to `#[payload]`.
+
. Drag a variable component and configure the following parameters:
** Set Name to `messageSid`.
** Set Value to `#[payload.getSid()]`.
+
. Add another transform message to create the input for the Redact Message operation:
+
[source,dataweave,linenums]
----
%dw 2.0
output application/java
---
{
    Body: "",
    From: payload.from,
    To: payload.'to'
}

Reply:
FYI, Transform message is the name of the component. I put "Transform message component." I changed "Variable" to "Set Variable component," also as shown in Studio.